openassessment/templates/openassessmentblock/oa_uploaded_file.html

```diff
@@ -12,8 +12,10 @@ <h5 class="{{ class_prefix }}__display__title">
 
     <div class="{{ class_prefix }}__display__file {% if not file_urls %}is--hidden{% endif %} submission__{{ file_upload_type }}__upload" data-upload-type="{{ file_upload_type }}">
         <div class="submission__answer__files">
-        {% for file_url, file_description, file_name in file_urls %}
-            <div class="submission__answer__file__block submission__answer__file__block__{{ forloop.counter0 }}" {% if not file_url %} deleted {% endif %}>
+        {% for file_url, file_description, file_name, file_size in file_urls %}
```

Comment:
Do we want to display file size to the user?

Reply:
I think we should, I also have it down at line 34

Reply:
I may have missed it, but the business requirements don't ask for it. Also, what is the value of that since we are only limiting the user to 500MB / single file. There is no cumulative limit.

Reply:
Oh, I totally forgot about that. You're right.